### DISCUSSION TITLE
How get add new address billing form template

### DISCUSSION THREAD

**Discussion by abanob0adel** [hallme/woo-address-book#139] [General]
hi how can get add new address form template i need add this in popup not new page

**Comment by matt-h** [comment]
I moved this to discussions since it isn't an issue with the plugin itself.

I'm not really sure how you'd pull out the new address template. We are only using the built in address form from WooCommerce on the My Account page and passing a query parameter to it to specify which address to edit/add. So it isn't something specific to this plugin.

If I were to start looking into building what you are looking for I'd start with looking in WooCommerce where the `wp-content/plugins/woocommerce/templates/myaccount/form-edit-address.php` template is called as that displays the form.
